debezium-core/src/main/java/io/debezium/relational/RelationalDatabaseConnectorConfig.java

```diff
@@ -349,6 +353,52 @@ public static DecimalHandlingMode parse(String value, String defaultValue) {
             .withInvisibleRecommender()
             .withDescription("The schemas for which events must not be captured (deprecated, use \"" + SCHEMA_EXCLUDE_LIST.name() + "\" instead)");
 
+    /**
+     * A comma-separated list of regular expressions that match database names to be monitored.
+     * Must not be used with {@link #DATABASE_BLACKLIST}.
+     */
+    public static final Field DATABASE_INCLUDE_LIST = Field.create(DATABASE_INCLUDE_LIST_NAME)
```

Comment:
Does it actually make sense to have that here? Most connectors rather work with the notion of "schemas", and connectors only connect to one particular database to begin with. I.e. which other connectors will use this?

Reply:
I think it makes sense to have the definition centralized even if currently used only by MySQL. SQL Server is another candidate